torch/_dynamo/skipfiles.py

```
@@ -160,7 +160,15 @@ def add(import_name: str):
     if isinstance(import_name, types.ModuleType):
         return add(import_name.__name__)
     assert isinstance(import_name, str)
-    module_spec = importlib.util.find_spec(import_name)
+    try:
+        module_spec = importlib.util.find_spec(import_name)
+    except AttributeError as e:
+        # handle fbgemm conflict caused by import torchrec.
+        if "'fbgemm' object has no attribute" in str(e):
+            return
+        raise
+    except ImportError:
+        return
```

Why are we now swallowing import error?